APMrover2/Rover.h

```diff
@@ -174,7 +174,7 @@ class Rover : public AP_HAL::HAL::Callbacks {
     AP_Baro barometer;
     Compass compass;
     AP_InertialSensor ins;
-    RangeFinder rangefinder{serial_manager, ROTATION_NONE};
```

Comment:
I shouldn't mention the serial_manager singleton, should I?

Reply:
That's actually not problematic, we stash the pointer (which is valid) and don't use it till init() so no race problems there (unless you are calling init before all the initializers run)